README.md

```diff
@@ -40,16 +42,52 @@ The exporter collects a number of statistics from the server:
 # TYPE memcached_current_connections gauge
 # HELP memcached_current_items Current number of items stored by this instance.
 # TYPE memcached_current_items gauge
-# HELP memcached_items_evicted_total Number of valid items removed from cache to free memory for new items.
-# TYPE memcached_items_evicted_total counter
-# HELP memcached_items_reclaimed_total Number of times an entry was stored using memory from an expired entry.
-# TYPE memcached_items_reclaimed_total counter
+# HELP memcached_evicted_total Number of valid items removed from cache to free memory for new items.
```

Comment:
Ah ok, the new ones have an additional `slab` label, so not exactly the same semantics...